tests/controllers_tests/controllers_test.go

```
@@ -366,16 +366,16 @@ func TestRetriveStatsBySourceIdDbPanic(t *testing.T) {
 			}
 		}
 	}()
-	cohortDefinitionControllerNeedsDb.RetriveStatsBySourceId(requestContext)
+	cohortDefinitionControllerNeedsDb.RetriveStatsBySourceIdAndTeamProject(requestContext)
 	t.Errorf("Expected error")
 }
 
-func TestRetriveStatsBySourceId(t *testing.T) {
+func TestRetriveStatsBySourceIdAndTeamProject(t *testing.T) {
 	setUp(t)
 	requestContext := new(gin.Context)
 	requestContext.Params = append(requestContext.Params, gin.Param{Key: "sourceid", Value: strconv.Itoa(tests.GetTestSourceId())})
 	requestContext.Writer = new(tests.CustomResponseWriter)
-	cohortDefinitionController.RetriveStatsBySourceId(requestContext)
+	cohortDefinitionController.RetriveStatsBySourceIdAndTeamProject(requestContext)
```

should fail since `teamproject parameter` is missing...